File: app/src/main/java/br/com/santander/testeandroid/investment/model/InvestmentResponse.java
package br.com.santander.testeandroid.investment.model;

import com.google.gson.annotations.SerializedName;

public class InvestmentResponse {
    @SerializedName("screen")
    private Screen screen;

    public Screen getScreen() {
        return screen;
    }

    public void setScreen(Screen screen) {
        this.screen = screen;
    }
}

File: SantanderInvestimento/app/src/main/java/br/com/santander/santanderinvestimento/investiment/data/entity/InvestmentResponse.kt
package br.com.santander.santanderinvestimento.investiment.data.entity

import br.com.santander.santanderinvestimento.investiment.domain.entity.Investment
import br.com.santander.santanderinvestimento.util.JSONConvertable
import com.google.gson.annotations.SerializedName


class InvestmentResponse(@SerializedName("screen") val screen: Investment) : JSONConvertable